### PR TITLE
get_session_key.R: fix PHP 8.1 major flaw

### PR DESCRIPTION
**Pull request by Jan-E**
TL;DR: param 'admin' for get_session_key succeeds in PHP 7.4, but fails in PHP 8.1. The param should be 'username'.

Limer sends 2 params for get_session_key: 'admin' and 'password'
```
body.json = list(method = "get_session_key",
                   id = " ",
                   params = list(admin = 'report',
                                 password = 'verysecret'))
```
https://github.com/cloudyr/limer/blob/master/R/get_session_key.R#L15-L18

Limesurvey expects params 'username' and 'password'
```
    /**
     * Set username and password by event
     *
     * @return null
     */
    public function remoteControlLogin()
    {
        $event = $this->getEvent();
        $this->setUsername($event->get('username'));
        $this->setPassword($event->get('password'));
    }
```
https://github.com/LimeSurvey/LimeSurvey/blob/971b1a72b5d3e62f299ef0a3829862f3d8ee5966/application/libraries/PluginManager/AuthPluginBase.php#L73-L83

I tested the fix in this PR against the same Limesurvey 5.3 instance, running PHP 7.4 and (symlinked) also PHP 8.1.13. See the comments under the commit: https://github.com/Jan-E/limer/commit/bc807c61fb834cb0034c376d814e80f126fc796c#commitcomment-94178825

Only afterwards I found that migliorati on the Limesurvey forum also suggested renaming 'admin' in 'username': https://forums.limesurvey.org/forum/installation-a-update-issues/127953-remotecontrol-doesn-t-work-after-update-to-5-3

Please review and merge.

### CODE REVIEW

**Comment by Jan-E** [comment]
@andrewheiss Could you merge this PR, please?

**Comment by Shnoulle** [comment]
> Limesurvey expects params 'username' and 'password'

Not really : https://github.com/LimeSurvey/LimeSurvey/blob/3909cf4bd2db67b3b0c6704d5daa99bedc880cee/application/helpers/remotecontrol/remotecontrol_handle.php#L3714-L3715

call here : https://github.com/LimeSurvey/LimeSurvey/blob/3909cf4bd2db67b3b0c6704d5daa99bedc880cee/application/helpers/remotecontrol/remotecontrol_handle.php#L47

It's event, not getRequest 

Stranget it was an issue here …

**Comment by Shnoulle** [comment]
I test a lot remoteControlLogin event because before only AuthDB work for plugin. Today i have Auth plugin working 

I do my test with AuthLDAP : https://github.com/LimeSurvey/LimeSurvey/commit/98c5293a01eb92cd4cc4930c9e667c3daa50cd44

If it work befpore (seems you have to fix limer only now) : something was update and broke system.

Please report the issue, and check with previous version of LimeSurvey.

If LS really wait for username then it broke with 7.4 … 


RemoteControl don't use [afterLoginFormSubmit](https://github.com/LimeSurvey/LimeSurvey/blob/f4233db80422ff9e40c8a6b7c79d96946f569d2a/application/libraries/PluginManager/AuthPluginBase.php#L63) but  [remoteControlLogin](https://github.com/LimeSurvey/LimeSurvey/blob/f4233db80422ff9e40c8a6b7c79d96946f569d2a/application/libraries/PluginManager/AuthPluginBase.php#L78)

if afterLoginFormSubmit event happen during Auth process for remoteControl it's a LimeSurvey issue.

**Comment by Shnoulle** [comment]
> `get_session_key() `works using a POST request which invokes the afterLoginFormSubmit event in the Authdb plugin.

get_session_key **don't** call afterLoginFormSubmit event. 

Only [remoteControlLogin](https://github.com/LimeSurvey/LimeSurvey/blob/f4233db80422ff9e40c8a6b7c79d96946f569d2a/application/helpers/remotecontrol/remotecontrol_handle.php#L3711)

**Comment by Jan-E** [comment]
Thanks a lot
